apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -184,6 +200,25 @@ export function InstructorAssessmentSettings({
                   ? resLocals.authz_data.has_course_permission_edit &&
                     !resLocals.course.example_course
                     ? html`
+                        <div>
+                          <button
+                            id="save-button"
+                            type="submit"
+                            class="btn btn-primary mb-2"
+                            name="__action"
+                            value="update_assessment"
+                          >
+                            Save
+                          </button>
+                          <button
+                            id="cancel-button"
```

Comment:
AFAICT this ID is never used; why include one?

Reply:
Copied this code over from the course settings editor. Removed this ID in [9173a63](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/9173a63860cec149d3796686882b0ec384a3dc16).